sympy/physics/control/lti.py

```diff
@@ -891,6 +896,13 @@ def __truediv__(self, other):
     def __neg__(self):
         return Series(TransferFunction(-1, 1, self.var), self)
 
+    def to_expr(self):
+        """Returns the equivalent ``Expr`` object."""
+        temp = []
+        for arg in self.args:
+            temp.append(arg.to_expr())
+        return Mul(*temp, evaluate=False)
```

Comment:
Using `temp` as a variable name is almost always a bad choice. The `temp` list isn't needed anyway:
```
return Mul(*(arg.to_expr() for arg in self.args))
```